### PR TITLE
Updates the working code base after 08_todo_service

### PR DESCRIPTION
**Pull request by brian-dlee**
Relates to #790 

This ticket updates the sample code that is referenced at the end of the Todo Service page. It's migration into the master branch should probably be coordinates with the updates to the documentation, but applying these changes in the same branch as the documentation update created a bit of a mess.

@kevgo

### CODE REVIEW

**Comment by brian-dlee** [comment]
I've found there are some code updates that are required to the `html-server` service, at the least.

**Comment by brian-dlee** [comment]
This branch needs some edits to the `html-server/service.yml` to accept and send the `todo` messages.

File: website/tutorial/part_2/part_2a/code_09/todo-app/todo/features/steps/steps.js
Comment (kevgo) [inline website/tutorial/part_2/part_2a/code_09/todo-app/todo/features/steps/steps.js:55]: Thanks for fixing this! Receiving is the correct word here!

File: website/tutorial/part_2/part_2a/code_09/todo-app/todo/features/update.feature
Comment (kevgo) [inline website/tutorial/part_2/part_2a/code_09/todo-app/todo/features/update.feature:44]: Haha!  :)

**Review by kevgo — APPROVED** [review]
Thanks for doing this, Brian! There are a ton of really good changes in here, that were long overdue (but I was way too busy to get to it).

**Comment by brian-dlee** [comment]
Oh, I can't take too much credit here! This is pretty much me running through the tutorial and imported the updating code base! 

I don't have write access to this repo so I can't merge the changes in my own. Can you help me out with that?

**Comment by kevgo** [comment]
One point for going the extra mile! I have given you write access, you should be able to merge future PRs yourself. Welcome to Team Exosphere!

**Pull request merged**
